src/shared/acl-util.c

```diff
@@ -232,7 +232,7 @@ int parse_acl(char *text, acl_t *acl_access, acl_t *acl_default, bool want_mask)
 
         split = strv_split(text, ",");
         if (!split)
-                return log_oom();
+                return -ENOMEM;
```

Comment:
Why shouldn't this log OOM any more?

The rest looks straightforward and good to me.